setup.py

```diff
@@ -42,6 +46,9 @@ def read(*filenames, **kwargs):
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```

Comment:
[pyroma](https://pypi.python.org/pypi/pyroma) complains if just the major versions are specified ([here's the check](https://bitbucket.org/regebro/pyroma/src/e05297042b1bdb15ca5938079da7c501185dee78/pyroma/ratings.py?at=default&fileviewer=file-view-default#ratings.py-141), which was in the initial version of the project).  Explicit minor versions are in the [classifier examples](https://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#classifiers) in the Python Packaging User Guide, but no explicit guidance about adding them.  It seems standard, but a quick PEP search didn't find guidance either way.

As for the list, I'm going from the Python versions tested in tox.  If that's expanded to more Python versions, it would make sense to add them to ``setup.py`` as well.